gtsfm/view_graph_estimator/view_graph_wo_filtering.py

```diff
@@ -0,0 +1,43 @@
+"""A ViewGraphEstimator implementation which is being run when the view_graph_estimator parameter
+is not provided in MultiViewOptimizer. This class only returns the graph input edges without any filtering.
```

Comment:
nit: we dont need to mention anything about MultiViewOptimizer here. Just saying that it doesn't do any filtering, and passes all the input edges should suffice.

Reply:
addressed [ddae6be](https://github.com/borglab/gtsfm/pull/569/commits/ddae6be1906674515722b8ad6a672428dd66bc20)